index.html

```diff
@@ -46,7 +54,8 @@ <h2 class="headline-section__sub-title">
 		<section class="work-section">
 			<div class="work-section__box">
 			<div class="work-section__card">
-				<div class="work-section__card--img"><img src="/images/SnapshootPortfolio.png" alt=""></div>
+				<div class="work-section__card--img mobile"><img src="/images/SnapshootPortfolio.png" alt="card snap"></div>
+				<div class="work-section__card--img desktop"><img src="./images/Card-images/SnapshootPortfolioDesktop1.png" alt="card snap"></div>
 				<div class="work-section__card--info">
```

Comment:
- [ ] Here, kindly please correct the font styling.  Especially the font-weight and the font-family for these elements on all these sections:

![image](https://user-images.githubusercontent.com/57421823/149450883-a3ec9515-0bca-45aa-b007-cda1f977a639.png)

That way, your project will look a lot more like the Figma 💪